week1/Q2.py

```diff
@@ -0,0 +1,51 @@
+class Node:
+	def __init__(self, value = None, next = None):
+		self.value = value
+		self.next = next
+
+
+class LinkedList:
+	def __init__(self):
+		self.first = None
+		self.last = None
+		self.length = 0
+
+	def clear(self):
+		self.__init__()
+
+	def add(self, x):
```

Comment:
The name `x` does not tell whether this is meant to be a node or a value. `value` would work better here.